test/integration/targets/azure_rm_managed_disk/tasks/main.yml

```diff
@@ -5,19 +5,19 @@
 
  - name: Clearing (if) previous disks were created (1/2)
```

Comment:
I know you didn't change this code but this can be simplified to 1 task like so

```
- name: Clearing (if) previous disks were created
  azure_rm_managed_disk:
    resource_group: "{{ resource_group }}"
    name: "{{ item }}"
    state: absent
  with_items:
  - "{{ managed_disk2 }}"
  - "{{ managed_disk1 }}"
```